front/client/page/categoryItem.test.js

```diff
@@ -0,0 +1,43 @@
+import { Selector } from 'testcafe'
+import VueSelector from 'testcafe-vue-selectors'
+
+fixture `ItemAddDelete`
+  .page `http://localhost:5000/haskell`;
+
+test('Add New Item to category', async addItem => {
+  const addItemBtn = Selector('.add-item-btn')
+  const addItemInput = Selector('input[aria-label="Item name"]')
+  const itemSubmitBtn = Selector('.add-cat-submit')
+  
+  await addItem
+    .click(addItemBtn)
+    .typeText(addItemInput, 'testcafe-item')
```

Comment:
Better use some unique name for test item for avoiding confilcts. Like with time or some UUID in it.

Reply:
there will be no conflicts, because next test is deleteItemTest which definitely deletes testcafe-item